modules/aws-api-gateway/main.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_apigatewayv2_api" "api_gateway" {
+  name          = "${var.environment}-${var.name}-api"
+  protocol_type = "HTTP"
+  description   = "API Gateway for various integrations"
+}
+
+resource "aws_apigatewayv2_stage" "api_gateway_stage" {
+  api_id = aws_apigatewayv2_api.api_gateway.id
+  name   = "$default"
```

Comment:
Should name be coming in as a variable? I've never seen this `"$default"` syntax before, do you have docs on what this is?